core/src/deno/script.rs

```diff
@@ -1,5 +1,5 @@
 use anyhow::Result;
-use deno_core::{op, Extension, JsRuntime, OpState, ZeroCopyBuf};
+use deno_core::{op, Extension, JsRuntime, OpState};
```

Comment:
A warning is generated at line 41, deprecation of op_declare
Tried to fix it for 10mns with no success -- I figure @spolu you can fix it in 1mn so asking for your help :bow:

Reply:
Will take a look :+1: (reviewing other PRs first)